src/server/game/Anticheat/AnticheatMgr.cpp

```diff
@@ -90,6 +90,81 @@ AnticheatMgr::~AnticheatMgr()
     m_Players.clear();
 }
 
+void AnticheatMgr::LoadBlockedLuaFunctions()
+{
+    uint32 oldmsTime = getMSTime();
+    auto pstmt = WorldDatabase.GetPreparedStatement(WORLD_SEL_ANTICHEAT_FUNCTIONS);
+    auto result = WorldDatabase.Query(pstmt);
+    if (!sWorld->getBoolConfig(CONFIG_LUABLOCKER_ENABLE))
```

Comment:
please move this block before doing any DB query as they are not needed if it's disabled

Reply:
https://github.com/TrinityCore/TrinityCoreCustomChanges/commit/1d1cdda4d4a033b99dde03f0de4606956be0f989